lib/connectors/sync_status.rb

```diff
@@ -26,7 +26,7 @@ class SyncStatus
       ERROR
     ]
 
-    PENDING_STATUES = [
```

Comment:
Lol, nice typo

Reply:
Yeah, I was thinking to leave it as an Easter Egg :)